src/main/kotlin/org/rust/ide/inspections/lints/RsLint.kt

```diff
@@ -16,81 +16,26 @@ import org.rust.lang.core.psi.ext.*
 enum class RsLint(
     val id: String,
     private val groupIds: List<String> = emptyList(),
-    val defaultLevel: RsLintLevel = WARN
+    private val defaultLevel: RsLintLevel = WARN
 ) {
+    // Rustc lints
+    // warnings
     NonSnakeCase("non_snake_case", listOf("bad_style", "nonstandard_style")),
```

Comment:
Were these "sorted" only by warning/error level, or did you intend to sort them by name? Since some of them are not sorted by name it seems.

Reply:
I sorted them only by the owner (rustc or clippy) and default lint level. Do you think we need to sort them by name as well?

Reply:
No I don't think so, the list is currently quite short. I was just wondering if it was intentional or not :)